scripts/irods/test/test_itree.py

```diff
@@ -66,6 +66,30 @@ def test_pattern_and_ignore_options_together__6791_6806(self):
         self.admin.assert_icommand(["itree",  "--regex-extended", "--ignore-regex", ".*[3-9]", "--pattern-regex", "ba(r|t)[[:digit:]]"],
                                     "STDOUT_MULTILINE", to_catch, use_regex = True)
 
+    def _verify_absolute_object_path(self, logical_path, must_exist):
+        self.assertEqual(logical_path[:1], '/', 'logical_path must start with "/"')
```

Comment:
I'm fine with that. I'll them out with a TODO for deletion on final round of reviews |& resolution of this conversation.  
It was only there as insurance, probably not needed.  Only if session_collection were NOT understood to be absolute

Reply:
Presently leaving in the assertion for the leading '/', since I considered that it might be an important consideration to the test (we want `itree` to do the right thing for both absolute and non-absolute paths).

If we decide in the end we don't need it, then I guess I'm for just deleting the helper fn and moving the existence / nonexistence assertions directly into the calling tests ?

Reply:
> Is this about protecting the test from future modifications?

I guess maybe it was ..? there's nothing in the name session_collection that guarantees it being absolute.  I thought this would make things more self-documenting, but am open to suggestions.

Reply:
There are several tests which rely on `session_collection` being an absolute path. If that changed suddenly, I think several tests would fail.

You don't have to guard against that at this time.